diff --git a/node_modules/easy-email-pro-theme/lib/index.js b/node_modules/easy-email-pro-theme/lib/index.js
index 6d34439..fde4313 100644
--- a/node_modules/easy-email-pro-theme/lib/index.js
+++ b/node_modules/easy-email-pro-theme/lib/index.js
@@ -11257,7 +11257,7 @@ const PreviewEmailDrawer = ({
           style: { height: "calc(100vh - 60px)", overflow: "hidden" }
         },
         /* @__PURE__ */ React__default.createElement(
-          "p",
+          "div",
           {
             style: {
               backgroundColor: "#ffffff",
